addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -370,3 +370,26 @@ def test_mailing_shortener(self):
                     link_info,
                     link_params=link_params,
                 )
+
+class TestMailingScheduleDateWizard(MassMailCommon):
+
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    @users('user_marketing')
+    def test_mailing_schedule_date(self):
+        mailing = self.env['mailing.mailing'].create({
```

Comment:
Would give at least a model, just to create a "real mailing" (model should always be given).

Reply:
mailing_model set to res_partner